guide/english/computer-science/databases/relational-databases/index.md

```diff
@@ -61,6 +61,37 @@ Figure 4 - Example of Pet table with owner field linked.
 
 A one-to-many relationship is one record linked to many other records, the example being the user Jerry having two pets. It could also be a many-to-many relationship where the tables could be books and authors, as authors could co-write many books. Finally the most common relationship type is one-to-one, a record that can only be linked to one, and only one, other record.
 
+### Foreign key
+
+To describe the Relationships above, Foreign keys(FK) should be be used to link table to tables.
```

Comment:
typo: ... should be **be** used ...
I think the end would be better as: `... used to link tables.`

Reply:
Thanks @nikrb!
I have update as your recommendation.